.github/workflows/linters.yml

```
@@ -103,7 +103,32 @@ jobs:
       - name: Environment Information
         run: npx envinfo
       - name: Lint JavaScript files
-        run: NODE=$(command -v node) make lint-js
+        run: |
+          set +e
+          NODE=$(command -v node) make lint-js
+          EXIT_CODE="$?"
+          if [ "$EXIT_CODE" != "0" ]
+          then
```

nit, for the sake of consistency
```suggestion
          if [ "$EXIT_CODE" != "0" ]; then
```